en/docs/administer/changing-to-embedded-derby.md

```diff
@@ -142,7 +142,7 @@ return via the datasource configurations by using the following options.
     Disabling the `          ConnectionRollbackOnReturnInterceptor         `
     is only possible with the **WUM** update and is effective from 14th
     September 2018 (2018-09-14). For more information on updating WSO2 API
-    Manager, see [Updating WSO2 Products](https://www.google.com/url?q=https%3A%2F%2Fdocs.wso2.com%2Fdisplay%2FADMIN44x%2FUpdating%2BWSO2%2BProducts&sa=D&sntz=1&usg=AFQjCNEMvqxxFtu8Qv8K4YugxNXrTfNtUA)
+    Manager, see [Updating WSO2 Products](https://docs.wso2.com/display/updates/Introduction)
```

Comment:
This page is under 'administer' folder. Check and correct in other relevant places as well.